pkg/engine/engine.go

```diff
@@ -228,12 +224,18 @@ func (e *QueryEngine) Execute(ctx context.Context, params logql.Params) (logqlmo
 	queueTime, _ := ctx.Value(httpreq.QueryQueueTimeHTTPHeader).(time.Duration)
 	stats := statsCtx.Result(durFull, queueTime, builder.Len())
 
+	tenants, err := tenant.TenantIDs(ctx)
```

Comment:
can there be more than one tenant in a context? I always assumed it was just one, and I use `user.ExtractOrgID` to get it